app/models/redirect.rb

```diff
@@ -0,0 +1,20 @@
+class Redirect
```

Comment:
I'd be tempted to call this class `ContentItem` since it doesn't necessarily represent a `Redirect` and is generic enough that it would work for any piece of content.

I'm also not sure about putting it in `app/models` because it's not an active record model, I would probably put it in `app/services` but that's just my personal preference.

Reply:
Good point. I renamed the class to ContentItem

ContentItem isn't what I would call a service though. I would normally have put it in a 'lib' directory or something generic but that doesn't exist here. The model 'Frequency' is already in the models directory and that is not an AR model so I didn't think ContentItem was too out of place in 'models'